apitools/gen/gen_client.py

```diff
@@ -211,6 +235,33 @@ def Run(self, _):
         _WriteInit(codegen)
 
 
+class GeneratePipPackage(appcommands.Cmd):
+
+    """Generate a client as a pip-installable tarball."""
+
+    def Run(self, _):
+        """Create a client in a pip package."""
+        discovery_doc = _GetDiscoveryDocFromFlags()
+        package = discovery_doc['name']
+        original_outdir = os.path.expanduser(FLAGS.outdir)
+        FLAGS.outdir = os.path.join(
+            FLAGS.outdir, 'apitools/clients/%s' % package)
+        FLAGS.root_package = 'apitools.clients.%s' % package
+        FLAGS.generate_cli = False
+        codegen = _GetCodegenFromFlags()
+        if codegen is None:
+            logging.error('Failed to create codegen, exiting.')
+            return 128
```

Comment:
this was actually a copy-paste from above, and was probably a bad idea where it appears above. ;) (i needed a safe exit code to use for catching certain errors in tests.)

switched to 1.